resources/sts/4.11/sts_installer_privatelink_permission_policy.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "ec2:ModifyVpcEndpointServiceConfiguration",
+		    "route53:ListHostedZonesByVPC",
+		    "route53:CreateVPCAssociationAuthorization",
+		    "route53:AssociateVPCWithHostedZone",
+		    "route53:DeleteVPCAssociationAuthorization",
+		    "route53:DisassociateVPCFromHostedZone",
+		    "route53:ChangeResourceRecordSets"
```

Comment:
Does this need to be here if its included in the core policy?

Reply:
Removed in https://github.com/openshift/managed-cluster-config/pull/1385.